src/main/java/de/tum/in/www1/artemis/web/rest/ImprintResource.java

```diff
@@ -0,0 +1,57 @@
+package de.tum.in.www1.artemis.web.rest;
+
+import javax.ws.rs.BadRequestException;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import de.tum.in.www1.artemis.domain.Imprint;
+import de.tum.in.www1.artemis.domain.enumeration.Language;
+import de.tum.in.www1.artemis.security.annotations.EnforceAdmin;
+import de.tum.in.www1.artemis.service.LegalDocumentService;
+
+/**
+ * REST controller for managing and retrieving the imprint.
+ */
+@RestController
+@RequestMapping("api/")
```

Comment:
admin is missing here
The file should probably also be called AdminImprintResource